cmd/k8s-operator/sts.go

```diff
@@ -46,15 +46,17 @@ const (
 	AnnotationHostname           = "tailscale.com/hostname"
 	annotationTailnetTargetIPOld = "tailscale.com/ts-tailnet-target-ip"
 	AnnotationTailnetTargetIP    = "tailscale.com/tailnet-ip"
+	AnnotationTailnetTargetFQDN  = "tailscale.com/tailnet-fqdn"
```

Comment:
Can we say MagicDNS somewhere? It's not just any fqdn (like Google.com) it's specifically a node name.

Reply:
Added a comment to this annot + containerboot env var comment. We can also say that in docs